plugins/quickinspector/quickinspector.cpp

```diff
@@ -262,6 +262,8 @@ QuickInspector::QuickInspector(ProbeInterface *probe, QObject *parent)
     filterProxy->addRole(ObjectModel::ObjectIdRole);
     probe->registerModel(QStringLiteral("com.kdab.GammaRay.QuickItemModel"), filterProxy);
 
+    connect(m_probe->probe(), SIGNAL(objectCreated(QObject*)),
```

Comment:
This can be conditional on m_probe->needsObjectDiscovery(), to avoid the extra performance cost when we are not attaching at runtime.